### ISSUE TITLE
pelayout tool is broken

### ISSUE THREAD

**Issue by jedwards4b** [ESMCI/cime#2573]
I would be happy to help fix this tool, but frankly I don't understand it.

**Comment by gold2718** [comment]
I can work on the documentation and fix the string format errors.
Can you help with the ESP thing? Why does it always get set to one?

**Comment by billsacks** [comment]
@gold2718 - regarding the documentation: From a quick glance, it looked pretty good to me in terms of the querying aspect, but an example would help in terms of the setting aspect. One other minor inconsistency between the help here and the (reworked) help of some other scripts is that others tend to put description and examples before the argument descriptions (see branch esmci:mvertens/documentation_updates - e.g., query_testlists, create_test). I don't think this HAS to be consistent, though, if you like it the way it is.

**Comment by gold2718** [comment]
@billsacks, @mvertens, @jedwards4b, I reworked the documentation but I have a question about the postscript (last part of the documentation block before the options section).
I do not recall why I originally put a copyright and license section in, but I notice that other user-facing tools do not have one.  Did they previously exist and were removed?
Should we consider having a standard statement, possibly added as an epilog (very end of help string)?

**Comment by jedwards4b** [comment]
I think that's a discussion that should involve our collaborators @rljacob

**Comment by gold2718** [comment]
Agree. Perhaps we should just point to the CIME LICENSE.TXT file?

**Comment by rljacob** [comment]
Pointing to LICENSE.TXT is fine but I also want the tools to be consistent so if the others aren't doing anything the pelayout shouldn't either.

**Comment by jedwards4b** [comment]
@rljacob I think that the question at hand is should we add a pointer to LICENSE.TXT in all user facing scripts?

**Comment by gold2718** [comment]
One idea is to include a master string (e.g., in CIME.utils):
```
license_string = """For information on Copyright and license, see
https://github.com/ESMCI/cime/blob/master/LICENSE.TXT"""
```
and then include in each script's `argparse.ArgumentParser` initializer:
```
epilog=license_string
```

**Comment by jedwards4b** [comment]
I think that we can put this in scripts/Tools/standard_script_setup.py and no change would need to be made elsewhere.

**Comment by jedwards4b** [comment]
I guess you still need to add the epilog to each script but license_string goes in scripts/Tools/standard_script_setup.py

**Comment by gold2718** [comment]
Okay, I pushed this method to PR #2586 using pelayout as an example.

**Comment by rljacob** [comment]
I don't think its a standard part of help strings to include license info.   So lets just leave it off.

**Comment by billsacks** [comment]
I agree with @rljacob that it doesn't seem standard to include license info in help strings, so would vote for leaving it off. But I don't have strong objections to including it if others feel we should.
